admin/admin-service/src/main/java/io/mosip/admin/service/impl/ApplicantDetailServiceImpl.java

```diff
@@ -0,0 +1,144 @@
+package io.mosip.admin.service.impl;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import io.mosip.admin.constant.ApiName;
+import io.mosip.admin.constant.ApplicantDetailErrorCode;
+import io.mosip.admin.packetstatusupdater.dto.ApplicantDetailsDto;
+import io.mosip.admin.packetstatusupdater.exception.DataNotFoundException;
+import io.mosip.admin.packetstatusupdater.exception.RequestException;
+import io.mosip.admin.packetstatusupdater.util.AuditUtil;
+import io.mosip.admin.packetstatusupdater.util.EventEnum;
+import io.mosip.admin.service.ApplicantDetailService;
+import io.mosip.admin.util.CbeffToBiometricUtil;
+import io.mosip.admin.util.RestClient;
+import io.mosip.admin.util.Utility;
+import io.mosip.biometrics.util.ConvertRequestDto;
+import io.mosip.biometrics.util.face.FaceDecoder;
+import io.mosip.kernel.core.idvalidator.exception.InvalidIDException;
+import io.mosip.kernel.core.idvalidator.spi.RidValidator;
+import io.mosip.kernel.core.util.JsonUtils;
+import org.apache.commons.codec.binary.Base64;
+import org.apache.commons.codec.binary.StringUtils;
+import org.json.JSONException;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.ResourceAccessException;
+
+import java.util.*;
+
+@Service
+public class ApplicantDetailServiceImpl implements ApplicantDetailService {
+
+    private static final String FACE = "Face";
+
+    @Autowired
+    RestClient restClient;
+
+    @Autowired
+    private RidValidator<String> ridValidator;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private AuditUtil auditUtil;
+
+    @Autowired
+    CbeffToBiometricUtil cbeffToBiometricUtil;
+
+
+    @Autowired
+    Utility utility;
+
+    private static final String IDENTITY = "identity";
+
+    private static final String RESPONSE = "response";
+
+    private static final String DOCUMENTS="documents";
+
+    private static final String ApplicantPhoto = "applicantPhoto";
+
+    private static final String VALUE = "value";
+    private static final String DOB = "dob";
+
+    @Value("${mosip.admin.applicantDetails}")
+    private String[] applicantDetails;
```

Comment:
Can we rename this config to mosip.admin.applicant-details.exposed-identity-fields ?

Reply:
taken care

Reply:
Resolved